### PR TITLE
Change example calls on calls report example

### PR DESCRIPTION
**Pull request by ren90**
Changed example calls on calls report response example
in order to give a more accurate description of call types.

### CODE REVIEW

File: reporting-api/README.md [hunk 1]
```diff
@@ -286,66 +286,66 @@ A `302 Redirect` response to an S3 link is returned. If your HTTP client automat
...
-      "abandon_time": 0,
-      "agent_speed_to_answer_time": 4,
-      "disposition_code": "Not Interested",
+      "total_time": 3,
+      "talk_time": 0,
+      "wait_time": 0,
+      "hold_time": 0,
+      "abandon_time": 3,
+      "agent_speed_to_answer_time": 0,
+      "disposition_code": null,
```
Comment (jcarvalho) [inline reporting-api/README.md]: What's the value of turning this into `null`?

Comment (ren90) [inline reporting-api/README.md]: Just copied a different call, can give that a value if its important

File: reporting-api/README.md [hunk 2]
```diff
@@ -286,66 +286,66 @@ A `302 Redirect` response to an S3 link is returned. If your HTTP client automat
...
+      "ivr_options": "",
+      "hangup": "offered",
       "is_in_business_hours": true,
-      "is_callback_from_queue": null,
+      "is_callback_from_queue": false,
       "is_transfer": false,
       "handling_agent_name": null,
       "handling_agent_email": null,
-      "recording_url": "https://recordings.talkdeskapp.com/calls/RENTc5YjJkYTViZTk3OWIwMDAzMDAwMDE5",
+      "recording_url": null,
```
Comment (jcarvalho) [inline reporting-api/README.md]: I'd keep this non-null as well, using the correct recordings URL format.

**Comment by jcarvalho** [comment]
In General, I think in this documentation we should strive to keep as many fields non-null as possible.

File: reporting-api/README.md (hunk 2 above)
Comment (jcarvalho) [inline reporting-api/README.md]: Hmm, scratch that, just realized this is an abandoned call :smile:

File: reporting-api/README.md [hunk 3]
```diff
@@ -286,66 +286,66 @@ A `302 Redirect` response to an S3 link is returned. If your HTTP client automat
...
-      "is_callback_from_queue": null,
+      "is_callback_from_queue": false,
       "is_transfer": false,
-      "handling_agent_name": null,
-      "handling_agent_email": null,
-      "recording_url": "https://recordings.talkdeskapp.com/calls/RENTc5YjJkODBiZTk3OWIwMDAzMDAwMDE4",
-      "is_external_transfer": false,
+      "handling_agent_name": "Charlie",
+      "handling_agent_email": "charlie@talkdesk.com",
+      "recording_url": "https://recordings.talkdeskapp.com/calls/RENTdmNjk5ZmU5YjhmMDMwMDAzMDAwMDIx",
```
Comment (jcarvalho) [inline reporting-api/README.md]: Any reason for using the legacy recording URL format?

Comment (ren90) [inline reporting-api/README.md]: that call came from like 6m ago, in QA. I'll change that, it makes sense

**Comment by ren90** [comment]
@jcarvalho 🎾

File: reporting-api/README.md [hunk 4]
```diff
@@ -280,72 +280,72 @@ A `302 Redirect` response to an S3 link is returned. If your HTTP client automat
...
+      "ivr_options": "",
+      "hangup": "offered",
       "is_in_business_hours": true,
-      "is_callback_from_queue": null,
+      "is_callback_from_queue": false,
       "is_transfer": false,
       "handling_agent_name": null,
       "handling_agent_email": null,
-      "recording_url": "https://recordings.talkdeskapp.com/calls/RENTc5YjJkYTViZTk3OWIwMDAzMDAwMDE5",
+      "recording_url": https://media-files.talkdeskapp.com/calls/CAa2c55bd75a3352f4244933b92942bf70/recordings/0.mp3,
```
Comment (jcarvalho) [inline reporting-api/README.md]: Missing `"` here.

File: reporting-api/README.md [hunk 5]
```diff
@@ -280,72 +280,72 @@ A `302 Redirect` response to an S3 link is returned. If your HTTP client automat
...
     {
-      "callsid": "CAbdc4e1549c50f11563addcde695c2303",
+      "callsid": "CA730a06a24e9ba8c55a4e793ba4f4e751",
       "type": "inbound",
       "start_at": "2016-03-29 10:22:02",
       "end_at": "2016-03-29 10:22:14",
       "talkdesk_phone_number": "+18443265295",
       "talkdesk_phone_display_name": "support",
       "contact_phone_number": "+17207081229",
-      "agent_name": "Charlie",
```
Comment (ricardoccpaiva) [inline reporting-api/README.md]: why drop this value ?

Comment (ren90) [inline reporting-api/README.md]: Because that call is was transferred to an external number

**Comment by ren90** [comment]
@ricardoccpaiva  🎾

File: reporting-api/README.md (hunk 5 above)
Comment (ricardoccpaiva) [inline reporting-api/README.md]: 👍

**Comment by ricardoccpaiva** [comment]
LGTM !

**Pull request merged**